src/templates/header.html

```diff
@@ -33,15 +33,25 @@
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <div class="navbar-nav mr-auto">
-            <a href="{{ url_for('.logout') }}" class="nav-item">
-                <span class="nav-link text-danger">Log Out</span>
-            </a>
+            {% if not hide_switch and show_staff %}
+                <a href="{{ url_for('.student_home') }}" style="margin-right: 10px;">
+                    <button class="btn btn-primary">Student</button>
+                </a>
+                <a href="{{ url_for('.staff_home') }}">
+                    <button class="btn btn-primary">Staff</button>
+                </a>
```

Comment:
These urls shouldn't be hard coded to student home or staff home. The point of this switch is actually to switch directly from staff's assignment page directly to student's assignment page in order to schedule a run. Quote task description from trello:

> When staff is on an assignment page, then click on this toggle, it should show the corresponding assignment page from student's perspective. (As opposed to the course page.)

You might want to use `request.endpoint` or `request.path` to achieve this.

Reply:
Ohh that makes sense. Thanks, I'll look into that!